modules/service/pricing-service.js

```diff
@@ -20,26 +20,30 @@ class PricingService {
             throw new Error('Calculate offer price method called. Holding time in minutes not defined!');
         }
 
-        const basePayoutCostInTrac = await this._calculateBasePayoutInTrac();
+        const {
+            basePayoutInTrac,
```

Comment:
Can you return the name of this variable to basePayoutCostInTrac, it was more expressive